runtime/streamlib-engine/tests/fixtures/known_audio_signal.py

```
@@ -0,0 +1,577 @@
+# Copyright (c) 2025 Jonathan Fontanez
```

Ruff (0.16.2)

[warning] The file is executable but no shebang is present

(EXE002)